src/neo-vm/ExecutionEngine.cs

```diff
@@ -217,7 +215,7 @@ private bool ExecuteInstruction()
                     // Push value
                     case OpCode.PUSH0:
                         {
-                            context.EvaluationStack.Push(EmptyBytes);
+                            context.EvaluationStack.Push(ReadOnlyMemory<byte>.Empty);
```

Comment:
This var could be static?

Reply:
`ReadOnlyMemory<byte>.Empty` is `static`.